CMakeLists.txt

```diff
@@ -136,6 +138,11 @@ if(WITH_GPU)
     endif(NOT WITH_DSO)
 endif(WITH_GPU)
 
+if(WITH_MKLDNN)
+    message(STATUS "MKLDNN_LIBRARY: ${MKLDNN_LIBRARY}")
```

Comment:
I can compile `mkldnn`. And after I manually download `mklml_lnx_2018.0.20170425.tgz`, I can compile `Paddle` with `mkldnn` too.